src/coreclr/tools/aot/ILCompiler.Compiler/Compiler/DependencyAnalysis/Target_X64/X64ReadyToRunHelperNode.cs

```diff
@@ -150,7 +150,6 @@ protected override void EmitCode(NodeFactory factory, ref X64Emitter encoder, bo
                         }
                         else
                         {
-                            ISymbolNode targetMethodNode = target.GetTargetNode(factory);
```

Comment:
Could you do the same fix for Target_ARM/Target_ARM64? They all have this copy-pasted.

Reply:
Righto, done.